src/BigBlueButton.php

```diff
@@ -418,14 +419,14 @@ public function hooksCreate(HooksCreateParameters $hookCreateParams): HooksCreat
         return new HooksCreateResponse($xml);
     }
 
-    public function getHooksListUrl(): string
+    public function getHooksListUrl(HooksListParameters $hooksListParameters): string
     {
-        return $this->urlBuilder->buildUrl(ApiMethod::HOOKS_LIST);
+        return $this->urlBuilder->buildUrl(ApiMethod::HOOKS_LIST, $hooksListParameters->getHTTPQuery());
     }
 
-    public function hooksList(): HooksListResponse
+    public function hooksList(HooksListParameters $hooksListParameters): HooksListResponse
```

Comment:
Hm, that is a breaking change, this parameter must not be a real parameter in 5.x and this method must trigger a deprecation notice when it is missing.